### PR TITLE
Links protocol enum

### PR DESCRIPTION
**Pull request by EllaMartirosyan**

### CODE REVIEW

File: src/common/models/protocol.enum.ts [hunk 1]
```diff
@@ -0,0 +1,17 @@
+export enum Protocol {
```
Comment (alebinson) [inline src/common/models/protocol.enum.ts:1]: Think **Protocol** --> **LinkType** is better

**Review by alebinson — CHANGES_REQUESTED** [review]
Think is worth to construct a linkType object  in this PR as well

File: src/common/models/protocol.enum.ts (hunk 1 above)
Comment (EllaMartirosyan) [reply src/common/models/protocol.enum.ts]: Fixed

**Review by alebinson — APPROVED** [review]
gr8!

**Pull request merged**